jsm.go

```diff
@@ -532,6 +532,10 @@ func (js *js) AddStream(cfg *StreamConfig, opts ...JSOpt) (*StreamInfo, error) {
 		return nil, ErrStreamNameRequired
 	}
 
+	if strings.Contains(cfg.Name, ".") {
```

Comment:
There is a function in the server.

```go
func isValidName(name string) bool {
	if name == "" {
		return false
	}
	return !strings.ContainsAny(name, ".*>")
}
```

Reply:
ok will open new PR to switch to that one